test/coverlet.core.performancetest/PerformanceTest.cs

```diff
@@ -14,8 +14,9 @@ namespace coverlet.core.performancetest
     /// </summary>
     public class PerformanceTest
     {
-        [Theory(Skip = "Only enabled when explicitly testing performance.")]
-        [InlineData(150)]
+        [Theory(/*Skip = "Only enabled when explicitly testing performance."*/)]
```

Comment:
Should the test be enabled by default now, or remain Skip in the repo code?

Reply:
The perf test is not run by default so we don't need the skip in place (avoids us changing the source just to test perf). That said I should have removed the skip instead of commenting it, I'll clean it up on my next change.